src/locales/en.ts

```diff
@@ -33,7 +33,14 @@ export default {
       phoneRequired: "Phone is required",
       cityRequired: "City is required",
       streetRequired: "Street is required",
-      postCodeRequired: "Post code is required"
-    }
+      postCodeRequired: "Post code is required",
```

Comment:
Just to let you know, you can use general translation like
```
...translations,
fieldRequired: '{0} is required'
```
And then you use it like this:

```
t("fieldRequired", ['Password'])
```

Your solution is good, this is for refrence only